template.yaml

```diff
@@ -2,11 +2,25 @@ kind: Template
 apiVersion: v1
 template: spark
 metadata:
-  name: spark
+  name: apache-spark
+  creationTimestamp: null
+  annotations:
+    description: Apache Spark with 3 worker nodes.
+    iconClass: icon-nodejs
+    tags: quickstart,Apache,Spark,apachespark
 labels:
   app: sparkcluster
 objects:
-
+- apiVersion: v1
+  kind: PersistentVolumeClaim
```

Comment:
i agree about making another template for the storage case. in general, i don't think we want to be advising folks towards attaching storage to their nodes, but i can see a use for it in some specific cases.